ironic/drivers/modules/drac/raid.py

```diff
@@ -470,7 +549,7 @@ def _find_configuration(logical_disks, physical_disks):
                                         physical_disks_by_type,
                                         free_space_mb))
             if not result:
-                error_msg = _('failed to find matching physical disks for all '
+                error_msg = _('failed to find matching physical disks for all'
```

Comment:
I think the space above needs to be added back in, otherwise there will be no space between "all" and "logical"

Reply:
Done